ngdart/lib/src/testability/js_api.dart

```diff
@@ -11,8 +12,8 @@ import 'package:meta/meta.dart';
 @anonymous
 abstract class JsTestability {
   external factory JsTestability({
-    @required bool Function() isStable,
-    @required void Function(void Function(bool didAsyncWork)) whenStable,
```

Comment:
Is there a reason why you removed the `didAsyncWork` parameter?

Reply:
Looking at `// TODO(b/168535057): Remove \`didWork\`.` comment, I delete them all, although it was necessary to delete only [there](https://github.com/angulardart-community/angular/blob/master/ngdart/lib/src/testability/js_impl.dart#L108). I can restore them.

Reply:
It's fine :)